server/mainapp/models.py

```diff
@@ -123,3 +123,183 @@ def fetch_data(self) -> response.JsonResponse:
             return docs
 
         raise DataFetchingError("There Are No Posts In The Database At This Moment")
+
+
+class Likes:
```

Comment:
To receive the likes, a URL will need to be created along with a view. This will refer to the endpoint that accepts the like and performs the database functionality - It will also update the UI with an increment in the Like Number on the screen.